### PR TITLE
get rid of can-util

### PR DESCRIPTION
**Pull request by cherifGsoul**
for https://github.com/canjs/canjs/issues/4218

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,14 +31,15 @@
   "dependencies": {
     "can-assign": "^1.1.1",
     "can-event-queue": "^1.0.0",
+    "can-key": "<2.0.0",
     "can-namespace": "1.0.0",
     "can-observation": "^4.0.0",
     "can-observation-recorder": "^1.0.0",
     "can-reflect": "^1.6.1",
     "can-simple-observable": "^2.0.0",
+    "can-single-reference": "<2.0.0",
```
Comment (chasenlehara) [inline package.json]: I’d write `^1.0.0` since the 1.0 has already been published.

Comment (cherifGsoul) [reply package.json]: done!

File: can-compute_test.js [hunk 2]
```diff
@@ -583,7 +583,8 @@ test("Listening to input change", function(){
 	});
 
 	input.value = 'foo';
-	domDispatch.call(input, "input");
```
Comment (chasenlehara) [inline can-compute_test.js]: Did this have to change? I would expect that you could use `can-dom-events.dispatch` for this.

Comment (cherifGsoul) [reply can-compute_test.js]: Yeah, if it is not changed the test will fail with ```TypeError: Cannot read property 'type' of undefined
    at Object.createEvent (http://localhost:8080/node_modules/can-dom-events/helpers/util.js:20:25)```

Comment (cherifGsoul) [reply can-compute_test.js]: The test passes with ```input.dispatch('input')``` and fails with ```domDispatch.call(input, "input")```
afer assigning the fn like this ``` var domDispatch = domEvents.dispatch```

Comment (chasenlehara) [reply can-compute_test.js]: Ah ok, so would this work?

```js
var domEvents = require("can-dom-events");

domEvents.dispatch(input, "input");
```

Comment (cherifGsoul) [reply can-compute_test.js]: I try it but it dosen't work!

Comment (cherifGsoul) [reply can-compute_test.js]: I change it to ```domEvents.dispatch(input, "input");```

**Pull request merged**